#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_faq.dpatch by Simo Kauppi <swk@nic.fi>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch removes the compilation and installation related
## DP: questions from the FAQ file.

@DPATCH@
diff -urNad swftools-0.9.0~/FAQ swftools-0.9.0/FAQ
--- swftools-0.9.0~/FAQ	2009-05-06 03:21:46.000000000 +0000
+++ swftools-0.9.0/FAQ	2009-05-06 03:25:03.000000000 +0000
@@ -9,81 +9,7 @@
     on http://www.swftools.org/donate.html
     .
 
-3.) How to compile/link swftools on Unix/Linux/BSD?
-
-    Download swftools-0.x.x.tar.gz. Then type, on the
-    command-line:
-
-    tar -zvxf swftools-0.x.x.tar
-    cd swftools-0.x.x
-    ./configure
-    make 
-    make install
-
-    (you will have to be root for the last step)
-
-4.) What libraries do I need to compile swftools on Unix/Linux/BSD?
-
-    You need freetype and jpeglib to compile pdf2swf.
-    Links:
-	freetype:
-	 http://www.freetype.org
-	jpeglib: 
-	 http://www.ijg.org/files/jpegsrc.v6b.tar.gz
-
-5.) when compiling the sourcecode version, configure doesn't find my jpeglib!
-
-    In case some of your libraries/include files are installed in /usr/local,
-    try the following:
-
-    rm -f config.cache
-    LDFLAGS="-L/usr/local/lib" CPPFLAGS="-I/usr/local/include" ./configure
-
-    Furthermore, a new installation of jpeglib (the following assumes it's in
-    /usr/local/lib) often requires doing a
-
-    ranlib /usr/local/lib/libjpeg.a
-    ldconfig /usr/local/lib
-
-    In order to install jpeglib, it's furthermore been remarked that you need
-    to do
-    make install-lib
-    make install-headers
-    .
-
-    If it still doesn't work, try removing the file "config.cache"
-    before running configure again.
-
-6.) There are problems during compilation:
-    Error messages complain about "undefined reference to `operator new(unsigned)'"
-    as well as "undefined reference to `__gxx_personality_v0'".
-
-    Try running configure as
-
-    LDFLAGS=-lstdc++ ./configure
-
-7.) How do I compile swftools on Mac OS X?
-
-    You need to get fink (http://fink.sourceforge.net), and install it.
-    After this, you need to install all the required libs (jpeg, freetype, lame):
-   
-    fink install libjpeg libjpeg-shlibs freetype2 freetype2-shlibs lame lame-shlibs
-
-    Then, you have to inform configure of where they are:
-
-    LDFLAGS="-L/sw/lib" CPPFLAGS="-I/sw/include -I/sw/include/lame" ./configure
-    make
-    sudo make install
-
-    If this doesn't work, especially if you want to use the python library and
-    get weird exceptions, try to remove all versions of jpeglib, freetype and 
-    lame from /usr/lib and /usr/local/lib, and then run configure / make / make install
-    again.
-
-    If you also want to use the python interface, you may have to copy the
-    Imaging headers (Imaging.h, ImPlatform.h, ImConfig.h) to
-      /System/Library/Frameworks/Python.framework/Versions/2.3/include/python2.3/
-    before running configure.
+3.-7.) Install/compile related FAQs are removed from the Debian package.
 
 8.) Is there a pdf2swf/avi2swf for Windows?
 
